Lab1.ipynb

```diff
@@ -1,11 +1,623 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The probability that the background produced the signal at x = 2.5 is 0.04393693362340742.\n"
+     ]
+    }
+   ],
+   "source": [
+    "pr = rayleigh.sf(2.5, 0, 1) # sf() is the survival function, it is defined as 1 - cdf\n",
+    "print(\"The probability that the background produced the signal at x = 2.5 is \"+ str(pr) + \".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A 3 E"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now I will convert this probability into an equivalent 'sigma' using the percent point function which is the inverse of cdf."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The equivalent 'sigma' of probability = 0.04393693362340742 is -1.7067212947438692.\n"
+     ]
+    }
+   ],
+   "source": [
+    "sr = stats.norm.ppf(pr) # ppf() is the percent point function which is the inverse of cdf\n",
+    "print(\"The equivalent 'sigma' of probability = 0.04393693362340742 is \" + str(sr) + \".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A 4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I will try various hypothetical measurement values below(x1 = 0.5; x2 = 1; x3 = 3.5) and see how the probabilities and 'sigmas' change.<br>\n",
+    "Depending on where my hypothetical measurement values are in the distribution, I use cdf() or sf() to calculate the probability correspond to each hypothetical measurement value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The probability that the background produced the signal at x = 0.5 is 0.1175030974154046,\n",
+      "and the equivalent 'sigma' of probability at x = 0.5 is -1.1875615473802972\n",
+      "\n",
+      "The probability that the background produced the signal at x = 1 is 0.6065306597126334,\n",
+      "and the equivalent 'sigma' of probability at x = 1 is 0.27028802073873587\n",
+      "\n",
+      "The probability that the background produced the signal at x = 3.5 is 0.002187491118182885,\n",
+      "and the equivalent 'sigma' of probability at x = 3.5 is -2.8497775017824143\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
I have the same comment as the one i made for line 272. We use integral to calculate the probability for continuous distribution so I think it might be better to say the range that the probability corresponds to rather than one value.